samwise.js

```diff
@@ -1,41 +1,41 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.